Stock.cc

```diff
@@ -1,24 +1,25 @@
 #include <iostream>
+#include<ctime>
+#include<cstdlib>
 #include "Stock.h"
 #include "json.hpp"
 
 using json = nlohmann::json;
 using namespace std;
 
-Stock::Stock() {
+Stock::Stock() {}
 
+void Stock::buyShares(int numSharesPurchased) {
+    numShares -= numSharesPurchased;
 }
 
-void Stock::buyShares(int numShares) {
-
+void Stock::sellShares(int numSharesSold) {
+    numShares += numSharesSold;
 }
 
-void Stock::sellShares(int numShares) {
-
-}
-
-void Stock::setMaxVariance(float maxVariance) {
-
+void Stock::setEODStockPrice() {
+    srand(unsigned(time(NULL)));
```

Comment:
The line `srand(unsigned(time(NULL)))` should only be run once to seed, can you put this in the ctor of the StockManager instead. Reference: https://stackoverflow.com/questions/5574914/srandtimenull-doesnt-change-seed-value-quick-enough